### ISSUE TITLE
Accessibility/Realm Settings/User Registration Tab: Serious Violation

### ISSUE THREAD

**Issue by agagancarczyk** [keycloak/keycloak#19365]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

admin/ui

### Describe the bug

Serious Violation was detected

![Screenshot 2023-03-27 at 20 07 26](https://user-images.githubusercontent.com/4890675/228042522-bf3031b2-5f5b-4883-9323-70626dc28647.png)

### Version

21.0.0

### Expected behavior

Violation not fixed

### Actual behavior

Violation fixed

### How to Reproduce?

USe Deque Axe Browser extension
